File: 2274-keep-multiplying-found-values-by-two/keep-multiplying-found-values-by-two.java
class Solution {
    public int findFinalValue(int[] nums, int original){
        int og=lin_search(nums,original);
        return og;
    }
    public int lin_search(int[] nums,int original)
    {
        int i;
        boolean flag=false;
        for(i=0;i<nums.length;i++)
        {
            if(nums[i]==original)
                flag=true;
        }
        if(flag)
            return original=lin_search(nums,2*original);
        else
            return original;
    }
}

File: 2274-keep-multiplying-found-values-by-two/keep-multiplying-found-values-by-two.py
class Solution(object):
    def findFinalValue(self, nums, original):
        """
        :type nums: List[int]
        :type original: int
        :rtype: int
        """
        if original not in nums:
            return original
        else:
            return self.findFinalValue(nums,2*original)

        